lib/publiccloud/k8s_provider.pm

```diff
@@ -0,0 +1,78 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Helper class for amazon connection and authentication
+#
+# Maintainer: Clemens Famulla-Conrad <cfamullaconrad@suse.de>, qa-c team <qa-c@suse.de>
```

Comment:
I guess we can just have qa-c contact, let's not put more burden on Clemens shoulders :)

Reply:
done